src/main/java/org/kuali/kfs/module/purap/document/PurchasingAccountsPayableDocumentBase.java

```diff
@@ -395,39 +363,33 @@ protected List getDeletionAwareUseTaxItems() {
     }
 
     /**
+     * @Override public List buildListOfDeletionAwareLists() {
```

Comment:
Can this commented-out version of buildListOfDeletionAwareLists() be removed?

Reply:
It's the same in base code so I think I will leave as as for now

Reply:
Ok, thanks for clarifying.